tests/ext/library_config/system_config_overloads_env.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
```

Comment:
Not related to this test but I am missing a test which shows the precedence between this and INI configurations